event/serializers.py

```diff
@@ -36,8 +36,7 @@ class EventCreateSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Event
-        fields = ('id', 'title', 'description', 'start_datetime', 'end_datetime',
-                  'organisation', 'cover', 'location', 'registration_url', 'sponser')
+        fields = ('__all__')
```

Comment:
@ZenabKhan why do we need `id` update here ?

Reply:
We need id here because if we removed id from here it won't returned the id of the newly created event. So, if you create a new event and you want the returned object of newly created event to be inserted to your already retrieved events then this won't work. Please correct me if I am wrong. @abdulmoizeng

Reply:
@usmandap  yes you are right 👍